flowgate-api/src/main/java/com/vmware/flowgate/service/AssetService.java

```diff
@@ -493,6 +570,11 @@ public static AssetIPMapping parseAssetIPMapingByString(String contentString) {
          data.setValueNum(value.getValueNum());
          data.setValue(value.getValue());
          data.setMetricName(value.getKey());
+         String unit = databaseUnitAndOutputUnitMap.get(value.getUnit());
```

Comment:
add comment that those kind of translation will be deprecated in next version.

Reply:
done